concordia/documents.py

```diff
@@ -15,18 +15,21 @@
 site_report = Index("site_reports")
 site_report.settings(number_of_shards=1, number_of_replicas=0)
 
+asset = Index("assets")
+asset.settings(number_of_shards=1, number_of_replicas=0)
+
 
 @user.doc_type
 class UserDocument(DocType):
     class Meta:
         model = User
 
-        fields = ["last_login", "date_joined", "username"]
+        fields = ["last_login", "date_joined", "username", "is_active"]
```

Comment:
Do we want to filter on this or restrict the set of users to `is_active=True`? Also, would we want to filter `is_staff=False`?

Reply:
We need to be able to tell how many users haven't activated their accounts, so we want everybody. We also want to be able to tell if any staff are in the top contributors, so we include them too.